presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -129,6 +129,7 @@
     private boolean legacyUnnestArrayRows;
 
     private boolean jsonSerdeCodeGenerationEnabled;
+    private int maxConcurrentMaterializations = 10;
```

Comment:
What about some smaller number , like 3 or 4 as default? :)

Reply:
I would keep it at 10. The default bucket count is 4096. Given 100MB per bucket we can still handle 4096 * 10 buckets. We don't want to regress much in wall time. So, i would restrain from decreasing the concurrency if there's no real need in it.

Reply:
ok, i'll switch it back